modules/devshell.nix

```
@@ -190,9 +190,12 @@ in
     motd = mkOption {
       type = types.str;
       default = ''
-        ${ansi.orange}🔨 Welcome to ${cfg.name}${ansi.reset}
+        {202}🔨 Welcome to ${cfg.name}{reset}
         $(type -p menu &>/dev/null && menu)
       '';
+      apply = builtins.replaceStrings
+        (map (key: "{${key}}") (attrNames ansi))
+        (map (esc: "${esc}") (attrValues ansi));
```

```suggestion
        (attrValues ansi);
```
That should be enough or did I miss something?

---

Heh. You're absolutely right I believe. Pretty obvious now that I look at it. It went through some iterations and this may be an artifact of that - just can't remember. I'll fix it.

---

@zimbatm So the attrNames still need to be mapped into the actual replacement strings.